cpp/open3d/core/EigenConverter.h

```diff
@@ -40,12 +40,16 @@ namespace eigen_converter {
 /// \brief Converts a Eigen matrix of shape (M, N) with alignment A and type T
 /// to a Tensor.
 ///
-/// \param matrix A templated Eigen matrix.
-/// \return A tensor converted from the Eigen matrix.
-template <class T, int M, int N, int A>
-core::Tensor EigenMatrixToTensor(const Eigen::Matrix<T, M, N, A> &matrix) {
-    core::Dtype dtype = core::Dtype::FromType<T>();
-    Eigen::Matrix<T, M, N, Eigen::RowMajor> matrix_row_major = matrix;
+/// \param matrix An Eigen matrix (e.g. Eigen::Matrix3f) or vector (e.g.
+/// Eigen::Vector3d).
+/// \return A tensor converted from the Eigen matrix. The resuliting Tensor is
```

Comment:
nit: `tensor`, `Tensor`, and `Eigen` might look inconsistent. May we change `tensor` to `Tensor`?

Reply:
done